hasher-matcher-actioner/hmalib/common/tests/test_image_sources.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+import io
+
+from PIL import Image
```

Comment:
do we need to add to `requirements-dev.txt` ?

Reply:
It is already there as part of threatexchange. No code here that is not already in pdq_hasher.py

Reply:
To check my understanding, if we remove it from python-threatexchange e.g. change the image processing lib this would then break here? (If so my gut is telling me it is worth having as a requirement then but I'll leave it up to you)

Reply:
Well, your gut makes sense to me. :P I'll add it.